tutor/templates/local/docker-compose.prod.yml

```diff
@@ -31,7 +31,7 @@ services:
     environment:
       SERVICE_VARIANT: lms
       DJANGO_SETTINGS_MODULE: lms.envs.tutor.production
-    command: celery --app=lms.celery worker --loglevel=info --hostname=edx.lms.core.default.%%h --max-tasks-per-child=100 --exclude-queues=edx.cms.core.default
+    command:{% for value in iter_lms_celery_parameters() %} {{value}}{% endfor %}
```

Comment:
Let's take the opportunity to convert this to a list, just like in k8s/deployments.yml.